benchmark.js

```diff
@@ -49,6 +49,8 @@ function main() {
   const flags = new Set(args.slice(1));
   const headless = !flags.delete('--no-headless');
   const ignoreHTTPSErrors = flags.delete('--ignore-https-errors');
+  ignoreHTTPSErrors && flags.add('--ignore-https-errors');
```

Comment:
No need for this check and to add the flag back. Just use `Set.has` rather than `Set.delete` to check if the set contains `--ignore-https-errors` but not delete it. I was using delete to remove flags that shouldn't be passed to chrome..

```javascript
const ignoreHTTPSErrors = flags.has('--ignore-https-errors');
```